lib/src/components/NxThreatCounter/__tests__/NxThreatCounter.test.tsx

```diff
@@ -13,41 +13,149 @@ describe('NxThreatCounter', function() {
+    });
+    const container = component.find('.nx-threat-counter-container');
+    expect(container).toExist();
+    expect(container.find('div').length).toBe(1);
+    const severeCounter = component.find('.nx-threat-counter--severe');
+    expect(severeCounter.find('.nx-threat-counter__count')).toHaveText('0');
+    expect(severeCounter.find('.nx-threat-counter__text')).toHaveText('Severe');
+  });
+
+  it('renders an .nx-threat-counter--moderate when moderateCount is specified', function() {
+    const component = getShallowComponent({
+      criticalCount: undefined,
+      severeCount: undefined,
+      moderateCount: 0,
+      lowCount: undefined,
+      noneCount: undefined
+    });
+    const container = component.find('.nx-threat-counter-container');
+    expect(container).toExist();
+    expect(container.find('div').length).toBe(1);
+    const moderateCounter = component.find('.nx-threat-counter--moderate');
+    expect(moderateCounter.find('.nx-threat-counter__count')).toHaveText('0');
+    expect(moderateCounter.find('.nx-threat-counter__text')).toHaveText('Moderate');
+  });
+
+  it('renders an .nx-threat-counter--low when lowCount is specified', function() {
+    const component = getShallowComponent({
+      criticalCount: undefined,
+      severeCount: undefined,
+      moderateCount: undefined,
+      lowCount: 0,
+      noneCount: undefined
+    });
+    const container = component.find('.nx-threat-counter-container');
+    expect(container).toExist();
+    expect(container.find('div').length).toBe(1);
+    const lowCounter = component.find('.nx-threat-counter--low');
+    expect(lowCounter.find('.nx-threat-counter__count')).toHaveText('0');
+    expect(lowCounter.find('.nx-threat-counter__text')).toHaveText('Low');
+  });
+
+  it('renders an .nx-threat-counter--none when noneCount is specified', function() {
+    const component = getShallowComponent({
+      criticalCount: undefined,
+      severeCount: undefined,
+      moderateCount: undefined,
+      lowCount: undefined,
+      noneCount: 0
+    });
+    const container = component.find('.nx-threat-counter-container');
+    expect(container).toExist();
+    expect(container.find('div').length).toBe(1);
+    const noneCounter = component.find('.nx-threat-counter--none');
+    expect(noneCounter.find('.nx-threat-counter__count')).toHaveText('0');
+    expect(noneCounter.find('.nx-threat-counter__text')).toHaveText('None');
+  });
+
+  it('renders all five indicators', function() {
     expect(getShallowComponent().find('.nx-threat-counter--critical')).toExist();
     expect(getShallowComponent().find('.nx-threat-counter--severe')).toExist();
     expect(getShallowComponent().find('.nx-threat-counter--moderate')).toExist();
     expect(getShallowComponent().find('.nx-threat-counter--low')).toExist();
+    expect(getShallowComponent().find('.nx-threat-counter--none')).toExist();
   });
 
   it('renders all of the values set in minimalProps', function() {
```

Comment:
This test and the one above it can be merged

Reply:
> This test and the one above it can be merged

Merged in https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/8cce8f0e7e32b56afa23137b9a7b1a8cb948a2bd